app/api-fetch/artist-by-id.ts

```
@@ -30,29 +43,61 @@
  */
 export default async function fetchArtistById(id: string) {
     console.log(`Fetching artist with ID: ${id}`);
-    try {
-        if (localStorage.getItem("artist-" + id) !== null && localStorage.getItem("artistTime-" + id) === null) {
+    if (localStorage) {
+        try {
+            if (localStorage.getItem("artist-" + id) !== null && localStorage.getItem("artistTime-" + id) === null) {
+                localStorage.removeItem("artist-" + id);
+            }
+            if (localStorage.getItem("artist-" + id) !== null || Date.now() - parseInt(localStorage.getItem("artistTime-" + id)!) < 3600000) {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)